index.html

```diff
@@ -0,0 +1,464 @@
+        </p>
+        <p class="note">
+          For example: if "some-new-behaviour" is added in the future a site developer can specify "[=route to/existing-client-navigate=]" to be the fallback with the following:
+        </p>
+        <pre class="note json">
+          {
+            "launch_handler": {
+              "route_to": ["some-new-behaviour", "existing-client-navigate"]
+            }
+          }
+        </pre>
+      </section>
+    </section>
+    <section>
+      <h2>
+        Handling Web App Launches
+      </h2>
+      <section>
+        <h2>
+          Launching a Web App
+        </h2>
+        <p>
+          When launching a web app via any means (e.g. OS launch, protocol handling, file handling, share target, link capturing, etc.) the trigger of the launch should run the steps to [=launch a web app=] with a {{LaunchParams}} containing relevant launch information (e.g. share data in the event of a share target launch).
+        </p>
+        <p>
+          The steps to <dfn>launch a web app</dfn> are given by the following algorithm and takes a {{LaunchParams}} |params:LaunchParams|.
+        </p>
+        <ol class="algorithm">
+          <li>If |params| is null, set |params| to a new {{LaunchParams}} with {{LaunchParams/targetURL}} set to [=manifest/start_url=].
+          </li>
+          <li>Set |client| to the result of running the steps to [=prepare a web app launch client=] passing [=manifest/route_to=] and |params|.{{LaunchParams/targetURL}}.
+          </li>
+          <li>Append |params| to the [=unconsumed launch params=] of the |client|'s document's {{Window.LaunchQueue}}.
+          </li>
+          <li>If the [=assigned launch consumer=] |consumer| is set:
+            <ol>
+              <li>[=list/For each=] |launch_params:LaunchParams| of [=unconsumed launch params=]:
+                <ol>
+                  <li>Invoke |consumer| with |launch_params|.
+                  </li>
+                </ol>
+              </li>
+              <li>Set [=unconsumed launch params=] to the empty list.
+              </li>
+            </ol>
+          </li>
+        </ol>
+      </section>
+      <section>
+        <h2>
+          Choosing a [=Web App Launch Client=]
+        </h2>
+        <p>
+          A <dfn>web app launch client</dfn> is a [=service worker client/window client=] associated with the web app.
+        </p>
+        <p class="note">
+          The exact form of this association is up to the user agent e.g. a dedicated app window separated in its UI presentation from general hyperlink browsing.
+        </p>
+        <p>
+          To <dfn>prepare a web app launch client</dfn>, given |route to| and |target URL:URL|:
```

Comment:
Instead of taking _route to_, you need to take an entire [processed manifest](https://www.w3.org/TR/appmanifest/#dfn-processed-manifest) here. Say "takes a [processed manifest](https://www.w3.org/TR/appmanifest/#dfn-processed-manifest) _manifest_".

Below when you refer to |route to|, instead write _manifest_["launch_handler"]["route_to"]".

Reply:
Done.